cookbook/teckin_sb50.rst

```diff
@@ -0,0 +1,83 @@
+TECKIN SB50 LED Bulb
+====================
+
+.. seo::
+    :description: ESPHome configuration for Teckin SB50 LED Bulb.
+    :image: images/teckin_sb50.jpg
+    :keywords: Teckin SB50
+
+.. figure:: images/teckin_sb50.jpg
+    :align: center
+    :width: 50.0%
+
+The Teckin SB50 Bulb's are a great option for lighting that can be flashed with tuya-convert. More details can be found at tuya-convert `github page <https://github.com/ct-Open-Source/tuya-convert>`__.
```

Comment:
I may not be 100% up to speed, but wasn't the problem tuya-convert used patched some time ago?

Reply:
Someone found a workaround for that as well, it was submitted in [PR 279](https://github.com/ct-Open-Source/tuya-convert/pull/279) and is now merged into the master. 
I have verified that it works with the Teckin SB50's, these were purchased from amazon on 11th October and flashed fine with tuya-convert.